stores/interface.go

```diff
@@ -0,0 +1,10 @@
+package stores
+
+import "customerApi/models"
+
+type Customer interface {
+	CreateCustomer(c models.Customer) error
+	GetCustomer(id string) (models.Customer, string)
+	UpdateCustomer(id string, c models.Customer) error
+	DeleteCustomer(id string) error
```

Comment:
id should be integer , and Methods are in customer interface, so need not add customer suffix with each name

Reply:
done